desktop/src/main.rs

```diff
@@ -898,6 +905,46 @@ fn winit_key_to_char(key_code: VirtualKeyCode, is_shift_down: bool) -> Option<ch
     })
 }
 
+/// Converts a `VirtualKeyCode` and `ModifiersState` to a Ruffle `TextControlCode`.
+/// Returns `None` if there is no match.
+fn winit_to_ruffle_text_control(
+    key: VirtualKeyCode,
+    modifiers: ModifiersState,
+) -> Option<TextControlCode> {
+    let shift = modifiers.contains(ModifiersState::SHIFT);
+    let ctrl_cmd = modifiers.contains(ModifiersState::CTRL)
+        || (modifiers.contains(ModifiersState::LOGO) && cfg!(target_os = "macos"));
```

Comment:
(Nit: Maybe add `// TODO: Extend this` considering the `TODO`s in https://github.com/ruffle-rs/ruffle/pull/2081 ?).